File: Code/java/circle.java
import java.util.Scanner;

class Main
{
    public static void main(String[] args)
    {
        float radius;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Radius: ");
        radius = scanner.nextFloat();

        System.out.println("Area: " + 3.14 * radius * radius);
        System.out.println("Circumference: " + 6.28 * radius);

        scanner.close();
    }
}

File: Code/cpp/circle.cpp
#include<iostream>
using namespace std;
int main()
{
    float radius;

    cout << "Enter radius: ";
    cin >> radius;

    cout << "Area: " << 3.14 * radius * radius;
    cout << "\nCircumference: "<< 6.28 * radius;
    return 0;
}